pages/common/tpp.md

```diff
@@ -1,8 +1,8 @@
 # tpp
 
-> Command-Line Based presentation tool.
+> Command-Line based presentation tool.
 
-- To use `tpp` you need to have a file which the `tpp` will interpret,you can find an example here:
+- To use tpp you need to have a file which the `tpp` will interpret,you can find an example here:
```

Comment:
- Why remove the back-ticks on the first `ttp` command here?
 - A comma after the space would make it look neater, though I would use a dash here personally.